docs/Guides/Recommendations.md

```diff
@@ -298,3 +299,36 @@ readinessProbe:
     timeoutSeconds: 3
     successThreshold: 1
     failureThreshold: 5
+```
+
+## Capacity planning for production
+<a id="capacity"></a>
+
+In order to rightsize your fastify application deployment in production, 
+you are highly recommended to perform your own measurements against 
+different configurations of the production environment 
+(2 vCPU, 1 vCPU, fractional parts of vCPU).
+
```

Comment:
Given this is a standalone section, we shouldn't limit this language to "vCPU". The user could be running on actual hardware or their environment could use different terminology.

```suggestion
In order to rightsize your Fastify application deployment in production, 
you are highly recommended to perform your own measurements against 
different configurations of the production environment, which may
use real CPU cores, virtual CPU cores (vCPU), or even fractional
vCPU cores. We will use the term vCPU throughout this
recommendation to represent any CPU type.

```